src/expr/impl/src/scalar/array.rs

```diff
@@ -134,10 +134,31 @@ fn map_access<'a>(
 /// select
 ///     map_contains(MAP{1:1}, 1),
 ///     map_contains(MAP{1:1}, 2),
-///     map_contains(MAP{1:1}, NULL::varchar),
-///     map_contains(MAP{1:1}, 1.0)
+///     map_contains(MAP{1:1}, NULL),
+///     map_contains(MAP{1:1}, '1'),
+///     map_contains(MAP{'a':'1','b':'2'}, 'ab');
 /// ----
-/// t f NULL f
+/// t f NULL t f
+///
+///
+/// query error
+/// select map_contains(MAP{1:1}, 1.0);
+/// ----
+/// db error: ERROR: Failed to run the query
+///
+/// Caused by these errors (recent errors listed first):
+///   1: Failed to bind expression: map_contains(MAP {1: 1}, 1.0)
+///   2: Bind error: Cannot check if numeric exists in map(integer,integer)
+///
+///
+/// query error
+/// select map_contains(MAP{1:1}, NULL::varchar);
+/// ----
```

Comment:
These are breaking change, but I assume it's fine since it's frontend-only change